src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -111,6 +112,22 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
   const isProtectedDegree = isDegreeWithProtectedFiles(publicationInstanceType);
   const canEditFiles = userCanEditRegistration(values) || userIsValidImporter(user, values);
 
+  function canEditFile(file: AssociatedFile) {
+    if (isProtectedDegree && isEmbargoed(file.embargoDate)) {
+      return !!user?.isEmbargoThesisCurator;
+    }
+
+    if (isProtectedDegree) {
+      return !!user?.isThesisCurator;
+    }
+
+    if (file.type === 'PublishedFile') {
+      return userCanUnpublishRegistration(values) ?? false;
+    }
+
+    return true;
```

Comment:
Uheldig å defaulte til `true`?
```suggestion
    return false;
```

Reply:
nvm, men skal såklart ha lov til å endre upubliserte filer!

Reply:
Enig i kommentaren din though! Man burde jo egentlig alltid defaulte til å ikke gi tilgang :)